doc/classes/EditorInterface.xml

```diff
@@ -285,6 +285,23 @@
 				See also [method Window.set_unparent_when_invisible].
 			</description>
 		</method>
+		<method name="popup_dialog_property_selector">
+			<return type="void" />
+			<param index="0" name="object" type="Object" />
+			<param index="1" name="callback" type="Callable" />
+			<param index="2" name="type_filter" type="PackedInt32Array" default="PackedInt32Array()" />
+			<description>
+				Pops up an editor dialog for selecting properties from [param object]. The [param callback] must take an Array[NodePath]. It is [method Callable.call_deferred] on an array containing the selection (see [method NodePath.get_as_property_path]) if an item is selected and [method Callable.call_deferred] on an empty array if the dialog is cancelled.
```

Comment:
I'm not sure `Array[NodePath]` in the description will be handled well by EditorHelp and by `make_rst.py`. Not sure if we have any way to refer to such types in descriptions. CC @dalexeev @YuriSizov

Reply:
I was able to find an instance in the documentation for `SceneReplicationConfig` of using an array of `NodePath`. 
```xml
		<method name="get_properties" qualifiers="const">
			<return type="NodePath[]" />
			<description>
				Returns a list of synchronized property [NodePath]s.
			</description>
		</method>
```
I can try and follow suit.

Reply:
So would you prefer I replace "Array[NodePath]" in my documentation with "NodePath[]" or "an [Array] of [NodePath]s?" Or something else all together?

Reply:
I went ahead with the second option.